python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py

```diff
@@ -55,6 +55,7 @@ def load_config():
 
 @pytest.yield_fixture(scope='class')
 def dirac_job(load_config):
+    from GangaDirac.Lib.Credentials.DiracProxy import DiracProxy
```

Comment:
Actually it won't work if that import is at the top - I think Ganga has to have setup the config first.